server/views/risk/riskManagementV1.pug

```diff
@@ -41,8 +42,11 @@ block content
 
           div.form-group
             label(for='riskManagementDetails') Provide details of your checks
-              span.form-hint.midMarginBottom.smallMarginTop You must confirm here that you have considered information from the police and social services when assessing the suitability of the proposed address.
-              span.form-hint.smallMarginBottom Also include details of information you're waiting for and when you should get it.
+              if version == '2'
```

Comment:
would v1 ever have `version == 2` ?

Reply:
Yikes - no sorry left in by mistake - fixing this now

Reply:
Sorted in latest commit